modules/140-user-authz/monitoring/prometheus-rules/user-authz/deprecated-cluster-authorization-rule-spec-limitnamespaces.tpl

```diff
@@ -0,0 +1,16 @@
+- name: kubernetes.user-authz.deprecated-spec
+  rules:
+  - alert: UserAuthzDeprecatedCARSpecLimitNamespaces
+    expr: >-
+      group by (module) (d8_deprecated_car_spec_limitnamespaces) == 1
+    labels:
+      severity_level: "9"
+    annotations:
+      description: |-
+        There is a cluster authorization rule with [deprecated]({{ include "helm_lib_module_uri_scheme" . }}://{{ include "helm_lib_module_public_domain" (list . "documentation") }}/modules/140-user-authz/#implementation-nuances) '.spec.limitNamespaces' parameter.
```

Comment:
The public domain may be empty. We agreed that, in this case, it is ok to redirect a user to the documentation website.